services/paymaster/src/constraint.rs

```diff
@@ -211,7 +214,9 @@ impl InstructionConstraint {
 #[derive(Serialize, Deserialize, Clone)]
 pub struct AccountConstraint {
     pub index: u16,
+    #[serde(default)]
     pub include: Vec<ContextualPubkey>,
```

Comment:
nit: I don't see the usecase of providing an AccountConstraint with empty include and exclude

Reply:
no such use case. just want to avoid having to specify empty arrays whenever possible.